core/services/keystore/master.go

```diff
@@ -141,17 +144,21 @@ func (ks *master) VRF() VRF {
 	return ks.vrf
 }
 
-func (ks *master) IsEmpty() (bool, error) {
-	var count int64
-	err := ks.orm.q.QueryRow("SELECT count(*) FROM encrypted_key_rings").Scan(&count)
-	if err != nil {
-		return false, err
-	}
-	return count == 0, nil
+type ORM interface {
+	isEmpty() (bool, error)
+	saveEncryptedKeyRing(*encryptedKeyRing, ...func(pg.Queryer) error) error
+	getEncryptedKeyRing() (encryptedKeyRing, error)
+}
+
+type keystateORM interface {
+	loadKeyStates() (*keyStates, error)
+	getNextNonce(common.Address, *big.Int, ...pg.QOpt) (int64, error)
+	incrementNextNonce(common.Address, *big.Int, int64, ...pg.QOpt) (int64, error)
```

Comment:
Are these still used?

Reply:
Oh I was thinking that #10108 had been merged. Does this become simpler after that PR?
- https://github.com/smartcontractkit/chainlink/pull/10108

Reply:
Yes, they're used from within `keystore.Eth()`.

Reply:
@jmank88 Ah, I hadn't seen that PR: it looks like keystateORM would only include the loadKeyStates method, so it is a simplification, yes.